PWGHF/TableProducer/CMakeLists.txt

```diff
@@ -64,6 +64,11 @@ o2physics_add_dpl_workflow(candidate-creator-omegac
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2::DCAFitter
                     COMPONENT_NAME Analysis)
 
+                    o2physics_add_dpl_workflow(candidate-creator-sigmac0plusplus
+                    SOURCES candidateCreatorSigmac0plusplus.cxx
+                    PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore ROOT::EG
```

Comment:
I believe ROOT::EG should be removed

Reply:
Removed, `O2::DCAFitter` put in place of it.